blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -36,7 +36,7 @@ export function pipelineBranchesUnsupported(pipeline) {
 
 const classicConfigLink = (pipeline) => {
     let link = null;
-    if (AppConfig.getInitialUser() !== 'anonymous') {
+    if (AppConfig.getInitialUser().id !== 'anonymous') {
```

Comment:
Yeah, will change to use that. Was going to to earlier actually and then just decided to leave it for now but since you say it too then ...